code-challenges.test.js

```diff
@@ -44,11 +61,27 @@ var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
 var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 // Expected output: [-823, 7, 23]
 
-
+//use a .push
 
 
 // b) Create the function that makes the test pass.
-
+describe("when an array of strings and numbers is called", ()=> {
+  it("returns an array of only odd numbers sorted from least to greatest", ()=> {
+    const sortArray(array){
+      let oddNums = [];
+      for(let i = 0; i < array.length; i++) {
```

Comment:
this is a great way to filter however there's another higher order function you can use for it 😄